src/renderer/src/app.tsx

```diff
@@ -76,6 +82,31 @@ declare module '@tanstack/react-router' {
 	}
 }
 
+let sentryEnvironment: SentryEnvironment = 'development'
+
+if (__APP_TYPE__ === 'release-candidate') {
+	sentryEnvironment = 'qa'
+} else if (__APP_TYPE__ === 'production') {
+	sentryEnvironment = 'production'
+}
+
+const sentryUserId = window.runtime.getSentryUserId()
+const enableSentry = window.runtime.enableSentryOnInit()
+
+initSentryElectron(
+	{
+		dsn: 'https://f7336c12cc39fb0367886e31036a6cd7@o4507148235702272.ingest.us.sentry.io/4509803831820288',
+		enabled: enableSentry,
+		// TODO: Enable tracing based on user consent in production
+		tracesSampleRate: sentryEnvironment === 'production' ? 0 : 1.0,
+		integrations: [tanstackRouterBrowserTracingIntegration(router)],
```

Comment:
I think we should only enable this in production with app usage metrics consent. I know the tracesSampleRate is 0 for production right now, but I think this also adds breadcrumbs to error reports, which some people might consider too much of a privacy issue without consent. That keeps is consistent with mobile.